.config.sh

```diff
@@ -22,6 +22,18 @@ if [[ ${PUBLIC_URL} == "https://searx.me" ]]; then
     PUBLIC_URL="http://$(primary_ip)/searx"
 fi
 
+# Architecture detection
+
+if [[ $(uname -m) == x86_64* ]]; then
+    export ARCH=amd64
+elif [[ $(uname -m) == i*86 ]]; then
+    export ARCH=386
+elif  [[ $(uname -m) == aarch64* ]]; then
+    export ARCH=arm64
+elif  [[ $(uname -m) == arm* ]]; then
+    export ARCH=armv6l
+fi
```

Comment:
Since it is about go we should better name this environment GO_ARCH I think.

There are already some go stuff defined in 

https://github.com/searx/searx/blob/f310305c54c3cd1d9fc74f09453294edbd2b5486/utils/lib.sh#L499-L501

I recommend to place the architecture detection there.

Reply:
Good idea I will move the architecture detection and rename the variable

Reply:
GO_ARCH or GOLANG_ARCH, what do you think?